### DISCUSSION TITLE
CELES

### DISCUSSION THREAD

**Discussion by tejasraman** [sebanc/brunch#1360] [Q&A] [answered]
My grandma has a CELES Chromebook (Samsung Chromebook 3) nearing its EOL. I plan to try and install Brunch on it (we are considering getting her a new laptop) using MrChromebox UEFI firmware. She likes to use Duolingo, and I wonder if the linux-chromebook 5.4 kernel has support for the internal mic of ALC5650(chtrt5650) Braswell devices like Celes?

**Comment by sebanc** [comment, marked as answer]
You can try the below unstable build with kernel 4.19 and "chromebook_audio" option:
https://github.com/sebanc/brunch-unstable/releases/tag/r96-unstable-20220102

**Comment by tejasraman** [comment]
Thanks so much @sebanc! I’ll try this when I get the chromebook 👍

**Comment by tejasraman** [comment]
Wait... is "chromebook_audio" a grub.cfg option?

**Comment by tejasraman** [comment]
@sebanc
I tried this on Brunch 101 with the Octopus 101 (for the longer update life 😉) and it did not work at all. Do you know why?

**Reply by tejasraman** [reply]
See my PR in brunch-unstable. Resolved
